.github/workflows/integration_test.yml

```diff
@@ -0,0 +1,39 @@
+# TODO: Support running all integration tests when it's possible to 
+# run self-hosted runners in parallel.
+name: Integration Test
+
+on:
+  pull_request:
+
+jobs:
+  changed_packages:
+    runs-on: [self-hosted, linux]
+    if: ${{ github.repository_owner == 'flutter-tizen' && github.event_name == 'pull_request' }}
```

Comment:
It seems to be an unnecessary condition.

Reply:
I've removed push on master trigger for now, so yes, thanks for the find.